pkg/app/pipectl/cmd/deployment/list.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deployment
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipecd/pkg/app/server/service/apiservice"
+	"github.com/pipe-cd/pipecd/pkg/cli"
+	"github.com/pipe-cd/pipecd/pkg/model"
+)
+
+type list struct {
+	root *command
+
+	statuses []string
+	appKinds []string
+	appIds   []string
+	appName  string
+	labels   []string
+	limit    int32
+
+	cursor string
+	stdout io.Writer
+}
+
+func newListCommand(root *command) *cobra.Command {
+	c := &list{
+		root:   root,
+		stdout: os.Stdout,
+	}
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Show the list of deployments.",
+		RunE:  cli.WithContext(c.run),
+	}
+
+	cmd.Flags().StringSliceVar(&c.statuses, "status", c.statuses, fmt.Sprintf("The list of waiting statuses. (%s)", strings.Join(model.DeploymentStatusStrings(), "|")))
+	cmd.Flags().StringSliceVar(&c.appIds, "app-id", c.appIds, fmt.Sprintf("The application id. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
+	cmd.Flags().StringSliceVar(&c.appKinds, "app-kind", c.appKinds, fmt.Sprintf("The kind of application. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
+	cmd.Flags().StringVar(&c.appName, "app-name", c.appName, "The application name.")
+	cmd.Flags().StringVar(&c.cursor, "cursor", c.cursor, "The cursor which returned by the previous request applications list.")
+	cmd.Flags().Int32Var(&c.limit, "limit", c.limit, "Upper limit on the number of return values. Default value is 100000.")
```

Comment:
The default value is zero, right?

Reply:
https://github.com/pipe-cd/pipecd/blob/add-pipectl-application-deployments/pkg/app/server/grpcapi/api.go#L462
I wrote 100000 because it was set at 100000 here. How about that?

Reply:
It is the default value of the API and should be treated separately from pipectl.
I feel it's too much to use it as a default value.

Reply:
Ok. I'll fix it.